README.md

```diff
@@ -22,7 +22,82 @@ Guidance on onboarding samples to docs.microsoft.com/samples: https://review.doc
 Taxonomies for products and languages: https://review.docs.microsoft.com/new-hope/information-architecture/metadata/taxonomies?branch=master
 -->
 
-Service Fabric can leverage the Managed Identity for Azure Active Directory (AD) to manage credentials in your application. These code examples show how to do so.
+This walkthrough steps through the process of deploying a service fabric cluster to Azure with managed identity enabled, and then deploying an application that uses managed identity to that cluster.

```

Comment:
This sample walks through, or this document walks through, or this is a walk through the process

Reply:
fixed